sdks/go/pkg/beam/runners/prism/internal/worker/worker.go

```diff
@@ -710,3 +660,130 @@ func (wk *W) MonitoringMetadata(ctx context.Context, unknownIDs []string) *fnpb.
 		},
 	}).GetMonitoringInfos()
 }
+
+type MultiplexW struct {
```

Comment:
Please add a comment about the purpose of MultiplexW.

If only because we'll get a lint comment about it on importing the code.